# disable_gcc_version_check1.patch by Thomas Hood
#
# Disable GCC version check in configure.in
# disable_gcc_version_check2.dpatch is the result of running autoconf
# after this patch has been applied. autom4te.cache can be deleted before
# patching.
Index: alsa-driver-1.0.15/configure.in
===================================================================
--- alsa-driver-1.0.15.orig/configure.in	2007-10-21 03:36:02.000000000 +0200
+++ alsa-driver-1.0.15/configure.in	2007-10-21 03:36:14.000000000 +0200
@@ -200,6 +200,7 @@
 AC_MSG_RESULT($kaversion)
 
 dnl Check for GCC version...
+if false; then
 AC_MSG_CHECKING(for GCC version)
 ac_save_CFLAGS="$CFLAGS"
 CFLAGS="$CFLAGS $KERNEL_INC $HACK_KERNEL_INC"
@@ -249,6 +250,7 @@
 kernel_compiler="")])
 CFLAGS="$ac_save_CFLAGS"
 my_compiler=`$CROSS_COMPILE$CC --version | head -n 1`
+fi
 dnl
 dnl Test section, uncomment any of these double lines to verify the bellow code:
 dnl
@@ -268,6 +270,7 @@
 dnl my_compiler="2.95.3"
 dnl
 
+if false; then
 AC_DEFUN([CHECK_COMPILER], [
   typevar=$2
   versionvar=$3
@@ -346,6 +349,7 @@
   fi
 fi
 fi
+fi
 dnl compiler version check
 
 AC_DEFUN([CHECK_KERNEL_HEADER], [
